src/ducks/keyStore.ts

```diff
@@ -0,0 +1,61 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import { KeyManager } from "@stellar/wallet-sdk";
+import { storePrivateKey, CreateKeyManagerResponse } from "helpers/keyManager";
+
+export const storePrivateKeyThunk = createAsyncThunk<
+  CreateKeyManagerResponse,
+  string,
+  { rejectValue: RejectMessage }
+>("keyManagerThunk", async (secret: string, { rejectWithValue }) => {
+  let result;
+  try {
+    result = await storePrivateKey(secret);
+  } catch (error) {
+    return rejectWithValue({
+      errorMessage: error.response?.detail || error.toString(),
+    });
+  }
+  return result;
+});
+
+interface RejectMessage {
+  errorMessage: string;
+}
+
+interface InitialState {
+  keyManager?: KeyManager;
```

Comment:
Is `KeyManager` serializable? It looks like a class instance, ideally we keep Redux state to plain objects

Reply:
@vcarl  It's not serializable. I was thinking we needed to store it for easier loading, but I guess we only need the ID and password, and then can just grab it from imports.

I'll change that, and then add a helper method for loading.